sw/device/tests/crypto/cryptotest/firmware/firmware.c

```diff
@@ -15,11 +15,13 @@
 #include "sw/device/tests/crypto/cryptotest/json/aes_sca_commands.h"
 #include "sw/device/tests/crypto/cryptotest/json/commands.h"
 #include "sw/device/tests/crypto/cryptotest/json/prng_sca_commands.h"
+#include "sw/device/tests/crypto/cryptotest/json/sha_sca_commands.h"
 
 // Include handlers
 #include "aes.h"
 #include "aes_sca.h"
 #include "prng_sca.h"
+#include "sha_sca.h"
```

Comment:
General comment (I know it's painful to change so let's first discuss): should we name this `sha3` instead of `sha` for disambiguation? Just for reference, HMAC provides SHA2 (and in the future potentially different versions of SHA2) and for OTBN there migth be a hardened SHA2 software implementation. In practice, SHA2 is often referred to as SHA. WDYT?

Reply:
Yeah makes sense, I've changed the code from SHA to SHA3. This gives us the possibility of having SHA2 in the future.